$DragonFly: src/gnu/usr.bin/diff/Attic/context.c.patch,v 1.1 2004/04/13 14:55:02 eirikn Exp $
--- ../diff.orig/src/context.c	2004-02-26 11:40:51.000000000 +0000
+++ context.c	2004-02-17 16:47:32.000000000 +0000
@@ -22,7 +22,7 @@
 
 #include "diff.h"
 #include <inttostr.h>
-#include <regex.h>
+#include <gnuregex.h>
 
 #ifdef ST_MTIM_NSEC
 # define TIMESPEC_NS(timespec) ((timespec).ST_MTIM_NSEC)
